studio/pages/account/security.tsx

```diff
@@ -0,0 +1,246 @@
+import { AddNewFactorModal } from 'components/interfaces/Account'
+import { AccountLayout } from 'components/layouts'
+import AlertError from 'components/ui/AlertError'
+import ConfirmationModal from 'components/ui/ConfirmationModal'
+import { FormHeader } from 'components/ui/Forms'
+import ShimmeringLoader from 'components/ui/ShimmeringLoader'
+import { useMfaListFactorsQuery } from 'data/profile/mfa-list-factors-query'
+import { useMfaUnenrollMutation } from 'data/profile/mfa-unenroll-mutation'
+import { useStore } from 'hooks'
+import { useState } from 'react'
+import { NextPageWithLayout } from 'types'
+import {
+  Accordion,
+  AlertDescription_Shadcn_,
+  AlertTitle_Shadcn_,
+  Alert_Shadcn_,
+  Badge,
+  Button,
+  IconAlertCircle,
+  IconAlertTriangle,
+  IconSmartphone,
+  IconTrash,
+  Modal,
+} from 'ui'
+
+const Security: NextPageWithLayout = () => {
+  const { data } = useMfaListFactorsQuery()
+
+  return (
+    <div className="1xl:px-28 mx-auto flex flex-col px-5 pt-6 pb-14 lg:px-16 xl:px-24 2xl:px-32">
+      <div className="flex items-center justify-between">
+        <FormHeader
+          title="Multi-factor authentication"
+          description="Add an additional layer of security to your account by requiring more than just a password to sign in."
+        />
+      </div>
+      <Accordion
```

Comment:
let's use `Collapsible` instead - refer to ProviderForm.tsx
the styling should be the same as that from the auth providers page